docs/bin/plugin_formatter.py

```diff
@@ -125,12 +125,13 @@ def write_data(text, options, outputname, module):
 #####################################################################################
 
 
-def list_modules(module_dir, depth=0):
+def list_modules(module_dir, depth=0, limit_to_modules="all"):
```

Comment:
Sheesh... So, before I get any deeper into this, the `limit_to_modules` parameter received after option parser is invoked should end-up producing one of the following values:

- `None` - no limitation, default if option is not specified.
- `[]` - don't build any modules. This should be set if you specify special keyword - `--limit-to-modules none`.
- `['a', 'b', 'c']` - build only specified module. This should be set if you specify `--limit-to-modules a,b,c`.

Three ways that I can see to go around this (with one simply not working):

1. Massage option value after call to `(options, args) = p.parse_args()`.

2. [Add a new type to option parser](https://docs.python.org/2/library/optparse.html#adding-new-types).

3. Use callback, but this won't work since we need to set type, and there is no `list` type (plus, we need `list + None` type in this particular case I guess).

Preferences? Or am I just overcomplicating this?